clangify.sh

```diff
@@ -1,11 +1,11 @@
 #!/bin/bash
 
-# This script will run clang-format on all non-3rd-party C++/Header files.
+# This script will run clang-format on all modified, non-3rd-party C++/Header files.
 
 set -e
 
-if hash clang-format 2>/dev/null; then
-	find . \( -name "*.cpp" -o -name "*.h" \) \
+if hash clang-format 2>/dev/null && hash git 2>/dev/null; then
+	git diff --name-only | xargs -I{} find '{}' \( -name "*.cpp" -o -name "*.h" \) \
```

Comment:
`git diff --name-only $(git merge-base origin/master)..HEAD` I think

Reply:
Looks to be `git diff --name-only $(git merge-base origin/master HEAD)`